.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+compute_engine_instance:
+  image_project: freebsd-org-cloud-dev
+  image: family/freebsd-13-1
+  platform: freebsd
+  disk: 100 # Gb
+
+build_task:
+  name: build
+  timeout_in: 120m
+  only_if: $CIRRUS_TAG != ''
+  env:
+    AWS_ACCESS_KEY_ID: ENCRYPTED[25d5e1d29570da70ceeb75e842b98be63d110824997b3c6b7bc0735b8acc686fbe0e9995cfe4d22a61d707a2e705d48b]
+    AWS_SECRET_ACCESS_KEY: ENCRYPTED[7fc23de7b958e671dcb485ff0baef777b83fb472f0afdde940122c994bc1586710d69b9d25855768bfd201904f3654dd]
+    S3_HOST: ENCRYPTED[d3fef1b5850e85d80dd1684370b53183df2218f2d36509108a2703371afd9ebd3f9596ad4de52487c15ea29baed606b7]
```

Comment:
Who holds the key for this? This is your S3 bucket, right?
To get this in and not let you pay for stuff, we'd need a HLS-AWS account, right?

Reply:
Yes, this is currently my own linode bucket. We can easily use a different bucket/service. That's what all the 3 secrets are for:

* S3_HOST
* AWS_ACCESS_KEY_ID
* AWS_SECRET_ACCESS_KEY